app/controllers/api/v1/businesses_controller.rb

```diff
@@ -59,7 +58,11 @@ def destroy
         def ratings
             @all_ratings = Business.find(params[:id]).reviews.pluck(:rating)
```

Comment:
I told someone else this, but these variables don't have to be instance variables (prepended by `@`). The reason the resource ('rating' in this case) is set to an instance variable is because the controllers were generated via scaffolding without rails being in 'api-only' mode. Rails sets the resource `@rating` because instance variables are accessible via the front end templates. If you wanted to reference `@all_ratings`, `@sum` and `@rating` in a front end template, this would be appropriate. Since you don't use _any_ rails front end templates, it's not. 
```
all_ratings = Business.find(params[:id]).reviews.pluck(:rating)
sum = all_ratings.reduce(:+)
@rating = sum.nil? ? 0 : Float(sum/all_ratings.length).ceil(1)
```